javascript/ql/src/semmle/javascript/dataflow/TaintTracking.qll

```diff
@@ -643,6 +643,60 @@ module TaintTracking {
 
   }
 
+  /**
+   * A less-than or greater-than expression
+   */
+  private class ExclusiveRelationalComparison extends RelationalComparison {
```

Comment:
Perhaps make this a predicate on `RelationalComparison` instead? I think it could be useful in other contexts.